crates/openvino/tests/util.rs

```diff
@@ -79,3 +82,13 @@ pub const DEFAULT_MARGIN: F32Margin = F32Margin {
 
 /// A helper type for manipulating lists of results.
 pub type Predictions = Vec<Prediction>;
+
+/// OpenVINO's v2024.2 release introduced breaking changes to the C headers, upon which this crate
+/// relies. This function checks if the running OpenVINO version is pre-2024.2.
+pub fn is_version_pre_2024_2() -> bool {
+    let version = version();
+    let mut parts = version.parts();
+    let year: usize = parts.next().unwrap().parse().unwrap();
+    let minor: usize = parts.next().unwrap().parse().unwrap();
+    year <= 2024 || (year == 2024 && minor < 2)
```

Comment:
@abrown Realized this half hour too late - this needs to be `year < 2024 ||`, was testing new version again and it skipped this test for `2024.4`.

Reply:
Ugh, yeah, that's right. I'll fix it.